.gitpod.yml

```diff
@@ -0,0 +1,10 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml)
+# and commit this file to your remote git repository to share the goodness with others.
+
+# Learn more from ready-to-use templates: https://www.gitpod.io/docs/introduction/getting-started/quickstart
+
+tasks:
```

Comment:
Gitpod is new to me. I'm not totally sure if we need to use it in this project. Can you help me understand how it fits here?

Reply:
[Gitpod](https://www.gitpod.io/) is a cloud environment that we can easily try any code base with no more waiting for dependencies to download and builds to finish. With gitpod I could run the project as following without downloading it into my local pc. 
<img width="1480" alt="Screenshot 2023-01-31 at 16 09 51" src="https://user-images.githubusercontent.com/47807830/215738077-37023119-6529-4996-8fc2-cec33b725235.png">